### ISSUE TITLE
How to perform combination of keys?

### ISSUE THREAD

**Issue by AdarshKumarGM** [octalmage/robotjs#467]
I have been using robotjs for performing single clicks as on date with the below code as shown in the example section:

`robot.keyTap("enter")`

However, I would to perform on a combination of keyboard keys such as "Command + Shift + h". I am little confused if the below code would help

`robot.keyTap("command")
           .keyTap("shift")
           .keyTap("h")`

or do I need to perform as below one:

`robot.keyTap("command","shift","h")`

Can you please provide a solution for this. I am not sure how to proceed from here. 

Basically, I am looking at performing for combination of keyboard keys in MacOS.

**Comment by bchoii** [comment]
https://robotjs.io/docs/syntax#keytapkey-modifier

To perform "Command + Shift + h",

`robot.keyTap("h", ["command", "shift"]);`

**Comment by oktapodia** [comment]
This issue has been closed because it has been inactive for > 180 days.
    You may reopen this issue if it has been closed in error.
